ts/components/__tests__/TosWebviewComponent.test.tsx

```diff
@@ -0,0 +1,169 @@
+import { fireEvent, render } from "@testing-library/react-native";
+import * as React from "react";
+import WebView from "react-native-webview";
+import renderer from "react-test-renderer";
+import * as O from "fp-ts/lib/Option";
+import {
+  WebViewErrorEvent,
+  WebViewMessageEvent,
+  WebViewNavigationEvent
+} from "react-native-webview/lib/WebViewTypes";
+import TosWebviewComponent from "../TosWebviewComponent";
+
+beforeAll(() => {
+  jest.resetAllMocks();
+});
+
+afterAll(() => {
+  jest.resetAllMocks();
+});
+
+describe("TosWebviewComponent", () => {
+  describe("The snapshot for the TosWebviewComponent", () => {
+    it("Renders correctlye", () => {
```

Comment:
What do you think about using a description in a forma like "it should .... with this prop"

```suggestion
    it("it should render correctly", () => {
```

Reply:
Yeah, that is definitely better